cmd/upgrade/executor/setup_job.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	//"k8s.io/klog"
 	"github.com/spf13/cobra"
-	//errors "github.com/openebs/maya/pkg/errors/v1alpha1"
```

Comment:
Please remove the imports which are not being used here.

Reply:
Will remove this in upcoming PRs.